themes/darcula.color-theme.json

```diff
@@ -121,6 +121,7 @@
                 "keyword.other.package.java",
                 "keyword.other.special-method.dockerfile",
                 "keyword.other.sql",
+                "keyword.other.rust",
```

Comment:
Keep the properties alphabetically sorted. This way it is easier to find them. There are other properties in this PR that need sorting as well.